common-api/src/main/java/org/pmiops/workbench/config/WorkbenchConfig.java

```diff
@@ -128,5 +128,6 @@ public static WorkbenchConfig createEmptyConfig() {
 
   public static class FeatureFlagsConfig {
     public boolean useBillingProjectBuffer = false;
+    public boolean unsafeAllowDeleteUser;
```

Comment:
Do we intend to eventually turn this on in production? If not, it probably doesn't qualify as a feature flag.

Reply:
Probably not. I wasn't sure where this should go though, do you have thoughts on where to put this flag?

Reply:
Per slack discussion, on further thought it's probably fine here; this tuple doesn't need to strictly apply to transient features. More generally a feature could be a permanent fixture which controls whether some functionality is enabled or disabled.